sdk/core/Azure.Core/aotcompatibility/AotCompatibilityTestApp.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PublishAot>true</PublishAot>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
+    <IsTestSupportProject>true</IsTestSupportProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Azure.Core.csproj" />
+    <ProjectReference Include="..\..\..\monitor\Azure.Monitor.OpenTelemetry.Exporter\src\Azure.Monitor.OpenTelemetry.Exporter.csproj" />
+
+    <TrimmerRootAssembly Include="Azure.Monitor.OpenTelemetry.Exporter" />
+
+    <!-- Update this dependency to its latest, which has all the annotations -->
+    <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="8.0.0-preview.7.23375.6" />
```

Comment:
Can we specify version in `eng/Packages.Data.props`?

Reply:
I'm going to delete this test project here but will capture this feedback for the CI pipeline I'm working on now.